Add reference to `continueUrlTarget` on bootstrap token payload (#63)
uphold/topper-docs: docs/flows/crypto-onramp.mdx

  - `fee` (_optional_): An object configuring the partner fee to be charged.
    - `percentage`: Percentage of the total source amount (the maximum allowed value is "5").
  - `continueUrl` (_optional_): URL to redirect the user to after the order is placed.

>   - `continueUrlTarget` (_optional_): How should topper redirect the user to the `continueUrl`.

Reviewer: ```suggestion
  - `continueUrlTarget` (_optional_): In which tab should the `continueUrl` be opened.
```

>     - `new-tab`: Redirect on a new tab.
>     - `same-tab`: Default value. Redirect on the same page/iframe
>     - `parent-tab`: Useful only for iframes, indicated to redirect on the page that contains the iframe.

Reviewer: ```suggestion
    - `new-tab`: Open in a new tab.
    - `same-tab`: Open in the same page or iframe (default).
    - `parent-tab`: Useful only in an iframe, open in the page that contains the iframe.
```
